src/App.js

```diff
@@ -2,12 +2,16 @@ import React from 'react';
 import './App.css';
 import { Container } from 'semantic-ui-react';
 import Map from './component/map/Map.js';
+import HelpButton from './component/help/HelpButton.js';
 
 function App() {
   return (
-    <Container>
-      <Map/>
-    </Container>
+    <React.Fragment>
```

Comment:
I saw you use React.Fragment, which is from React. I previously used Container, which is from semanticUI. I wonder which one we should keep. My motivation using container is that it's like the React.Fragment in Semantic but might be easier to style. What do you think?

Reply:
sorry nevermind, ignore my pervious comment.